data/json/recipes/recipe_armor.json

```diff
@@ -2569,6 +2569,56 @@
   ]
 },{
   "type" : "recipe",
+  "result": "survivor_goggles",
+  "category": "CC_ARMOR",
+  "subcategory": "CSC_ARMOR_HEAD",
+  "skill_used": "fabrication",
+  "skills_required" : ["tailor", 4],
+  "difficulty": 5,
+  "time": 20000,
+  "reversible": false,
+  "autolearn": true,
+  "qualities" : [
+    {"id":"CUT","level":1,"amount":1}
+  ],
+  "tools": [
+      [
+      [ "needle_bone", 10 ],
+      [ "needle_wood", 10 ],
+      [ "tailors_kit", 10 ],
+      [ "sewing_kit", 10 ]
+    ],
+    [
+      [ "welder", 28 ],
+      [ "soldering_iron", 28 ],
+      [ "toolset", 28 ]
+      ]
+  ],
+  "components": [
+      [
+      [ "goggles_ski", 1 ],
+      [ "goggles_welding", 1 ],
```

Comment:
Welding goggles seem a little over-dark for making sunglasses, since sunglasses won't suffice for welding.